### PR TITLE
Add EntityManager.unwrap() test to JPA Spec 2.0 FAT bucket, add Eclip…

### PR DESCRIPTION
**Pull request by jgrassel**
Add Eclipselink and JPA 21 Spec JPA FAT Buckets
Add EntityManager bucket to JPA 20 Spec Bucket (disabled until Eclipselink fix [Eclipselink bug 547173] is delivered)
Added FAT utility MBean to `componenttest-1.0` as a tool to enrich server-side test components (MBean's only ability currently is to list installed features, which is used by the EntityManager test case to skip jpa-2.2 feature until the fix is delivered into that feature as well.)

### CODE REVIEW

**Comment by jgrassel** [comment]
#build

**Comment by jgrassel** [comment]
Adding @jhanders34 as a reviewer since this introduces new eclipse projects.

File: dev/com.ibm.ws.jpa_eclipselink_fat/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+javac.source: 1.8
+
+src: \
+	fat/src,\
+	test-applications/helpers/DatabaseManagement/src,\
+	test-applications/example/src,\
+	test-framework/src
+
+fat.project: true
+tested.features: jdbc-4.0, beanvalidation-1.1, jpa-2.0, el-3.0
```
Comment (dazey3) [inline dev/com.ibm.ws.jpa_eclipselink_fat/bnd.bnd]: Discussed this and it seems removing this might be in order

**Comment by jhanders34** [comment]
I examined the bnd and eclipse files and things looked good to me.  From looking at the other files, it would seem that any new files that you are adding in this PR should have a copyright of 2019, not 2018.

File: dev/com.ibm.ws.jpa_eclipselink_fat/bnd.bnd (hunk 1 above)
Comment (jgrassel) [reply dev/com.ibm.ws.jpa_eclipselink_fat/bnd.bnd]: Removed.

**Pull request merged**